master/internal/core.go

```diff
@@ -791,6 +793,67 @@ func (m *Master) restoreNonTerminalExperiments() error {
 	return nil
 }
 
+func (m *Master) restoreGenericTasks(ctx context.Context) error {
+	snapshots := []command.CommandSnapshot{}
```

Comment:
style: use var syntax for making zero-valued things: `var snapshots []command.Command`